app/src/userpages/components/ProductsPage/index.jsx

```diff
@@ -112,9 +112,9 @@ const Actions = (product: Product) => {
     const redirectToPublishProduct = useCallback((id: ProductId) => (
         dispatch(push(formatPath(links.marketplace.products, id, 'publish')))
     ), [dispatch])
-    const redirectToProduct = useCallback((id: ProductId) => (
-        dispatch(push(formatPath(links.marketplace.products, id)))
-    ), [dispatch])
+    const redirectToProduct = useCallback((id: ProductId) => {
+        window.open(formatPath(links.marketplace.products, id), '_blank', 'noopener')
```

Comment:
Let's go with that, sure. I'd love to make the "View product" item a regular link with `target="_blank"`. This is how our tile structures look like right now:

<img width="855" alt="Screenshot 2020-02-21 14 23 06" src="https://user-images.githubusercontent.com/320066/75038297-65632980-54b6-11ea-8183-879841f9e445.png">

There's a ticket for tiles lined up – [`PLATFORM-1255`](https://streamr.atlassian.net/browse/PLATFORM-1255). I made comment about this particular problem.